recipes/genologics/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version="0.3.12.post0" %}
+
+package:
+  name: genologics
+  version: "{{ version }}"
+
+source:
+  fn: genologics-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/b4/04/bd023ff57246cd5724aec7fca289d2da519aa3c96cffac8e3a239f073c1e/genologics-{{ version }}.tar.gz
+  md5: 1812a2f27873c230d5a9bc86419bba8a
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - requests
+
+  run:
+    - python 2.6.*|2.7.*  # [py2k]
```

Comment:
In BioConda we build automatically against py27,py35, py36, see here: https://github.com/bioconda/bioconda-recipes/blob/bulk/scripts/env_matrix.yml#L1

So this is not needed :)